psa_builder.py

```diff
@@ -112,6 +120,9 @@ def _is_git_installed():
     elif _is_make_installed() != 0:
         logging.error('"Make" is not installed. Exiting...')
         return -1
+    elif _is_srec_installed() != 0:
+        logging.error('"srec_cat" is not installed. Exiting...')
```

Comment:
Could you please update the readme with how to get srec_cat and to list it as a requirement?

Ensure the Vagrant bootstrap script also installs srec_cat if it isn't installed by default already.

Reply:
I will update the vagrant file, but which readme are we talking about which lists as a requirement?

Reply:
If there is no README file that lists requirements, no need to take any action. The Vagrant bootstrap can be used to see what the run-time requirements are.